src/scripts/farming/Plot.ts

```diff
@@ -47,35 +438,63 @@ class Plot implements Saveable {
         }
 
         this.isUnlocked = json['isUnlocked'] ?? this.defaults.isUnlocked;
-        this.boosted = json['boosted'] ?? this.defaults.boosted;
         this.berry = json['berry'] ?? this.defaults.berry;
-        this.timeLeft = json['timeLeft'] ?? this.defaults.timeLeft;
+        this.age = json['age'] ?? this.defaults.age;
+        this.mulch = json['mulch'] ?? this.defaults.mulch;
+        this.mulchTimeLeft = json['mulchTimeLeft'] ?? this.defaults.mulchTimeLeft;
     }
 
     toJSON(): Record<string, any> {
         return {
             isUnlocked: this.isUnlocked,
-            boosted: this.boosted,
             berry: this.berry,
-            timeLeft: this.timeLeft,
+            age: this.age,
+            mulch: this.mulch,
+            mulchTimeLeft: this.mulchTimeLeft,
         };
     }
 
-    // Knockout getters/setters
-    get isUnlocked(): boolean {
-        return this._isUnlocked();
+    /**
+     * Finds the plot indices that are around the plot in a 3x3 square
+     * @param index The plot index
+     * @param filter An optional filter callback for filtering out indices
+     */
+    public static findNearPlots(index: number, filter?: (n: number) => boolean): number[] {
+        const plots = [];
+
+        const colIdx = index % 5;
+        const rowIdx = (index - colIdx) / 5;
```

Comment:
These 5s could do with being constants somewhere, and also replaced wherever else is assuming that the farm is a 5x5